pkg/cmd/dictionary/entries/browse/browse.go

```diff
@@ -0,0 +1,238 @@
+	Words         []string `json:"words,omitempty"`
+	Decomposition string   `json:"decomposition,omitempty"`
+	ObjectID      string
+	Language      string
+}
+
+const (
+	// CustomEntryType is the type of a custom entry in a dictionnary (i.e. added by the user).
+	CustomEntryType EntryType = "custom"
+	// StandardEntryType is the type of a standard entry in a dictionnary (i.e. added by Algolia).
+	StandardEntryType EntryType = "standard"
+)
+
+var (
+	// DictionaryNames returns the list of available dictionnaries.
+	DictionaryNames = func() []string {
+		return []string{
+			string(search.Stopwords),
+			string(search.Compounds),
+			string(search.Plurals),
+		}
+	}
+)
+
+// NewBrowseCmd creates and returns a browse command for dictionnaries' entries.
+func NewBrowseCmd(f *cmdutil.Factory, runF func(*BrowseOptions) error) *cobra.Command {
+	cs := f.IOStreams.ColorScheme()
+
+	opts := &BrowseOptions{
+		IO:           f.IOStreams,
+		Config:       f.Config,
+		SearchClient: f.SearchClient,
+		PrintFlags:   cmdutil.NewPrintFlags().WithDefaultOutput("json"),
+	}
+	cmd := &cobra.Command{
+		Use:       "browse {<dictionary>... | --showStopWords}",
+		Args:      cobra.OnlyValidArgs,
+		ValidArgs: DictionaryNames(),
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return DictionaryNames(), cobra.ShellCompDirectiveNoFileComp
+		},
+		Short: "Browse dictionary entries",
+		Long: heredoc.Docf(`
+			This command retrieves all entries from the specified %s dictionnaries.
+		`, cs.Bold("custom")),
+		Example: heredoc.Doc(`
+			# Retrieve all entries from the custom dictionary (doesn't include default stopwords)
+			$ algolia dictionary entries browse stopword
+
+			# Retrieve all entries from the "stopword" and "plural" dictionnaries
+			$ algolia dictionary entries browse stopword plural
+
+			# Retrieve all entries from all dictionnaries
+			$ algolia dictionary entries browse --all
+
+			# Retrieve all entries from the "stopword" dictionnaries (including default stopwords)
+			$ algolia dictionary entries browse stopword --showDefaultStopwords
+		`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.All && len(args) > 0 || !opts.All && len(args) == 0 {
+				return cmdutil.FlagErrorf("Either specify dictionaries' names or use --all to browse all dictionaries")
+			}
+
+			if opts.All {
+				opts.Dictionnaries = []search.DictionaryName{search.Stopwords, search.Plurals, search.Compounds}
+			} else {
+				opts.Dictionnaries = make([]search.DictionaryName, len(args))
+				for i, dictionnary := range args {
+					opts.Dictionnaries[i] = search.DictionaryName(dictionnary)
+				}
+			}
+
+			return runBrowseCmd(opts)
+		},
+	}
+
+	cmd.Flags().BoolVarP(&opts.All, "all", "a", false, "browse all dictionnaries")
+	cmd.Flags().BoolVar(&opts.ShowDefaultStopwords, "showDefaultStopwords", false, "browse dictionaries and include default stopwords")
```

Comment:
We usually use kebab-case for the flags. Also, how about a more generic name for this flag, like `--include-defaults` or `--include-stopwords-defaults` ? A shorthand version would also be welcomed :)